rearrange/procthor_rearrange/utils.py

```diff
@@ -0,0 +1,162 @@
+from typing import Dict, Union, List, Any, Optional, Sequence
+
+import ai2thor.controller
+import lru
+from allenact_plugins.ithor_plugin.ithor_environment import IThorEnvironment
+from allenact_plugins.ithor_plugin.ithor_util import include_object_data
+
+from rearrange.utils import hand_in_initial_position
+
+_UNIFORM_BOX_CACHE = {}
+
+
+class ObjectInteractablePostionsCache:
```

Comment:
Should we be using this version of the `ObjectInteractablePostionsCache` also for the old rearrangement code?

Reply:
I adapted it a bit for that purpose 👍